### DISCUSSION TITLE
Generate

### DISCUSSION THREAD

**Discussion by SebastianNikken30** [sa-si-dev/virtual-select#163] [Q&A]
Hello, I would like to know how to make it so that none of the selected items are brought to me, because when I open the modal and the multiselect, all the selected items appear.

**Comment by sa-si-dev** [comment]
> Hello, I would like to know how to make it so that none of the selected items are brought to me, because when I open the modal and the multiselect, all the selected items appear.

@SebastianNikken30 sorry, I am not getting your questions. Could you please give me an example to explain your question?
